Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -301,221 +317,131 @@ deqms_results %>%
-           large_effect=abs(!!sym(xcol))>logFC_threshold) %>%
-    mutate(label=ifelse(sig & large_effect, gsub('(.*GN=| PE=.*)', '', Master.Protein.Descriptions), NA)) %>%
-    ggplot(aes(!!sym(xcol), -log10(P.Value), colour=sig)) +
-    geom_point(aes(size=sig)) +
-    theme_camprot(base_size=15, border=FALSE) +
-    scale_colour_manual(values=c('grey', get_cat_palette(1)), guide=FALSE) +
-    scale_size_manual(values=c(0.5, 2), guide=FALSE) +
-    xlab(xlabel) +
-    ylab('-log10(p-value)') +
-    theme(strip.background=element_blank()) +
-    ggrepel::geom_label_repel(aes(label=label), show.legend=FALSE)
-}
-
-
-volcano_plot <- function(lmResults,
-                         adj.P.Val_threshold=0.05,
-                         logFC_threshold=2,
-                         xcol='logFC',
-                         xlabel){
-  lmResults %>%
-    mutate(sig=adj.P.Val<adj.P.Val_threshold,
-           large_effect=abs(!!sym(xcol))>logFC_threshold) %>%
-    mutate(label=ifelse(sig & large_effect, gsub('(.*GN=| PE=.*)', '', Master.Protein.Descriptions), NA)) %>%
-    ggplot(aes(!!sym(xcol), -log10(P.Value), colour=sig)) +
-    geom_point(aes(size=sig)) +
-    theme_camprot(base_size=15, border=FALSE) +
-    scale_colour_manual(values=c('grey', get_cat_palette(1)), guide=FALSE) +
-    scale_size_manual(values=c(0.5, 2), guide=FALSE) +
-    xlab(xlabel) +
-    ylab('-log10(p-value)') +
-    theme(strip.background=element_blank()) +
-    ggrepel::geom_label_repel(aes(label=label), show.legend=FALSE)
-}
-
-
-
-volcano <- volcano_plot(lmResults,
-                          adj.P.Val_threshold=0.1,
-                          logFC_threshold=0.9,
-                          xlabel='Effect of Toxic ASO on RNA binding (log2)')
-
-#define function build_results_list
-build_results_list <- function(lmResults, design, results, volcano){
-  list('data'=lmResults,
-       'design'=design,
-       'results'=results,
-       'volcano'=volcano)
-}
-
-lmResults <- build_results_list(lmResults,
-                                   design,
-                                   lmResults,
-                                   volcano)
-print(lmResults$volcano)
-```
 
 ---------------------------------------------------
 Let's try looking at any differences between my L2K and safe ASO
 
 ```{r}
 #L2k vs Safe ASO statistical testing
 
-data_to_model2 <- expInputRBP[,pData(expInputRBP)$Treatment!='Toxic'] # or some equivalent filtering to remove the samples which are Vehicle treatment
+
+# Remove the samples which are Toxic ASO treated
+data_to_model2 <- expInputRBP[,pData(expInputRBP)$Treatment!='Toxic'] 
 
 #assign input and RBPs as factors in the material column of the data_to_model dataset.
 material2 <- factor(data_to_model2$material, levels=c('Input', 'RBP'))
 
 safe <- factor(data_to_model2$Treatment, levels=c('Safe', 'Vehicle'))
 
-
 design2 <- model.matrix(~material2*safe)
 
 # And then usual lines to perform linear modeling with limma...
 ```
 
-Build the run_limma function
```

Comment:
No need to redefine the `run_limma` function. The original one was generalised enough to handle different input